docs/materials/htc/part2-ex1-files.md

```diff
@@ -85,7 +85,7 @@ for word in sorted(words.keys()):
 1.  Download the input file for the script (263K lines, ~1.4 MB) and save it in your submit directory:
 
         :::console
-        username@learn $ wget http://proxy.chtc.wisc.edu/SQUID/osgschool19/mon-2.1-words.txt
+        username@learn $ wget http://proxy.chtc.wisc.edu/SQUID/osgschool20/mon-2.1-words.txt
```

Comment:
Ooh, does this exist?

Reply:
This is probably going to conflict with Jason’s PR now. I may need help here.

Reply:
Yup! Somebody much more clever than I already cp'ed last year's squid stuff over. 

I'll take a look at the conflict in a few.